gdg_events/android_study_jam.md

```diff
@@ -5,7 +5,7 @@
 
 - Google Developers Study Jams is a free series of global, community-run, in-person study groups. 
 - The first Study Jams series started in 2015. The 2016 series officially starts between February 1, 2016 and May 1, 2016, and is presented in partnership with Udacity. 
-- Google Developer Group facilitators -- consisting of members within the GDG community itself and, and have gone through the curriculum prior to the start of the series -- lead the Study Jams that meet at least once or on a weekly basis, between 1 February 2016 to 1 May 2016.
+- Google Developer Group facilitators -- consisting of members within the GDG community itself and have gone through the curriculum prior to the start of the series -- lead the Study Jams that meet at least once a week or rather on a weekly basis, between 1 February 2016 to 1 May 2016.
```

Comment:
Okay


*Stanley Ndagi /** Andela.com
<http://andela.com/?utm_source=andela&utm_medium=email&utm_campaign=signature> *
*Software Engineer / 0718 154504 <+254718154504> / @Andela
<http://bit.ly/andelatwitter>*

*"The startup that's harder to get into than Harvard" - *CNN
<http://snip.ly/HfId>
See what The Wall Street Journal <http://snip.ly/u9AI> has to say.

On Mon, Feb 8, 2016 at 11:16 AM, Friedger Müffke <notifications@github.com>
wrote:

> In gdg_events/android_study_jam.md
> <https://github.com/gdg-wisdom/wisdom-2016/pull/11#discussion_r52140838>:
>
> > @@ -5,7 +5,7 @@
> >
> >  - Google Developers Study Jams is a free series of global, community-run, in-person study groups.
> >  - The first Study Jams series started in 2015. The 2016 series officially starts between February 1, 2016 and May 1, 2016, and is presented in partnership with Udacity.
> > -- Google Developer Group facilitators -- consisting of members within the GDG community itself and, and have gone through the curriculum prior to the start of the series -- lead the Study Jams that meet at least once or on a weekly basis, between 1 February 2016 to 1 May 2016.
> > +- Google Developer Group facilitators -- consisting of members within the GDG community itself and have gone through the curriculum prior to the start of the series -- lead the Study Jams that meet at least once a week or rather on a weekly basis, between 1 February 2016 to 1 May 2016.
>
> The study jam could be a one time event or a weekly event serie
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-wisdom/wisdom-2016/pull/11/files#r52140838>.
>